.pre-commit-config.yaml

```diff
@@ -48,18 +48,6 @@ repos:
     - id: blacken-docs
       additional_dependencies: [black==23.1.0]
 
--   repo: https://github.com/asottile/yesqa
-    rev: v1.4.0
-    hooks:
-    - id: yesqa
```

Comment:
Oops, forgot to enable the matching RUFF check.

Reply:
Is that RUF100 and PGH004 ? I couldn't figure it out... Ruff says it can replace `yesqa` but doesn't provide guidance on what that looks like.

Reply:
It's RUF100. PGH replaces pygrep-hooks, which looks for blanket noqa (no code).